content/en/cloud_cost_management/container_cost_allocation.md

```diff
@@ -16,79 +16,93 @@ further_reading:
+3. For AWS ECS support, set up [**Datadog Container Monitoring**][3] in ECS tasks.
      - Optionally, enable [AWS Split Cost Allocation][5] for usage-based ECS allocation.
 
 ## Cost allocation
 
-Cost allocation divides EC2 compute and EBS volume costs in the [Cost and Usage Report][4] (CUR) into associated individual tasks or pods. These divided costs are then enriched with tags from nodes, pods, tasks, and volumes. This lets you break down costs by any associated dimensions.
+Cost allocation divides host compute and AWS EBS volume costs in the bill into associated individual tasks or pods. These divided costs are then enriched with tags from nodes, pods, tasks, and volumes (AWS only). This lets you break down costs by any associated dimensions.
 
 ### Kubernetes
 
 #### Compute
 
-For Kubernetes compute allocation, a Kubernetes node is joined with its associated EC2 instance costs. The node's cluster name and all node tags are added to the entire EC2 compute cost for the node. This allows you to associate cluster-level dimensions with the cost of the instance, without considering the pods scheduled to the node.
+For Kubernetes compute allocation, a Kubernetes node is joined with its associated host instance costs. The node's cluster name and all node tags are added to the entire compute cost for the node. This allows you to associate cluster-level dimensions with the cost of the instance, without considering the pods scheduled to the node.
 
 Next, Datadog looks at all of the pods running on that node for the day. The cost of the node is allocated to the pod based on the resources it has used and the length of time it ran. This calculated cost is enriched with all of the pod's tags.
 
 **Note**: Only _tags_ from pods and nodes are added to cost metrics. To include labels, enable labels as tags for [nodes][7] and [pods][8].
 
-#### Persistent volume storage
+#### Persistent volume storage (AWS only)
 
 For Kubernetes Persistent Volume storage allocation, Persistent Volumes (PV), Persistent Volume Claims (PVC), nodes, and pods are joined with their associated EBS volume costs. All associated PV, PVC, node, and pod tags are added to the EBS volume cost line items.
 
 Next, Datadog looks at all of the pods that claimed the volume on that day. The cost of the volume is allocated to a pod based on the resources it used and the length of time it ran. These resources
 include the provisioned capacity for storage, IOPS, and throughput. This allocated cost is enriched with all of the pod's tags.
 
+#### Agentless Google Kubernetes costs
 
-### ECS on EC2
+In some cases, you may wish to view the costs of GKE clusters without having Datadog infrastructure monitoring enabled. In these cases CCM supports viewing costs allocated using [GKE cost allocation](https://cloud.google.com/kubernetes-engine/docs/how-to/cost-allocations), with some limitations.
```

Comment:
idk but needs some editing, maybe from docs team 🙏 

```suggestion
CCM supports GKE clusters that do not have Datadog infrastructure monitoring enabled, with some limitations. Enable [GKE cost allocation](https://cloud.google.com/kubernetes-engine/docs/how-to/cost-allocations) on unmonitored GKE clusters to access this limited feature set.
```

Reply:
What are the limitations? Can you specify? Otherwise I recommend removing reference to limitations if it's easily identified by the user.
```suggestion
To view the costs of GKE clusters without enabling Datadog infrastructure monitoring, use [GKE cost allocation](https://cloud.google.com/kubernetes-engine/docs/how-to/cost-allocations). Enable GKE cost allocation on unmonitored GKE clusters to access this feature set.
```